wordpress-mobile/WordPress-iOS
# Stats Today extension: use the user-selected site

Fixes #6538 

This fixes the issue where the user selected site was not being used for the Stats Today widget.

To test:

**Prerequisite**: In `FeatureFlag.swift`, `return false` for `statsRefresh`.

---
- Select a non-primary site.
- Go to Stats > Today > Use this site.
- Go to the device's notification center, or long press on the WP icon.
- Verify the Stats widget is using the newly selected site.

---
- Swipe to close the WP app.
- Rerun the WP app.
- Go to the device's notification center, or long press on the WP icon.
- Verify the Stats widget is still using the selected site.

---
![selected_site_used](https://user-images.githubusercontent.com/1816888/50801539-76e17600-12a2-11e9-9bac-690f6e244b7d.png)

---




Update release notes:
